src/dialog-renderer.ts

```diff
@@ -128,7 +144,7 @@ export class DialogRenderer implements Renderer {
   private setupClickHandling(dialogController: DialogController): void {
     this.stopPropagation = e => { e._aureliaDialogHostClicked = true; };
     this.closeDialogClick = e => {
-      if (!dialogController.settings.lock && !e._aureliaDialogHostClicked) {
+      if (dialogController.settings.backdropDismiss && !e._aureliaDialogHostClicked) {
```

Comment:
`backdrop` - I went over some other dialogs/modals impls and this word was used
`dismissableOverlay` - for me this conveys that the overlay can be dismissed